### DISCUSSION TITLE
Make a second label

### DISCUSSION THREAD

**Discussion by Ndmanh123** [clientIO/joint#1749] [Q&A] [answered]
i would like to create a another label in header in joint.shapes.standard.HeaderedRecord like this 
![image](https://user-images.githubusercontent.com/104048582/180729893-52c156be-7fb8-4582-ace8-4929bf280a17.png)
cant i make this and how do i make this 
thank for the help

**Comment by jamesgeorgewilliams** [comment, marked as answer]
You can extend predefined shapes in JointJS or create custom elements.
https://resources.jointjs.com/tutorial/custom-elements
https://resources.jointjs.com/tutorial/ts-shape


For the **standard.HeaderedRecord**, you could create some custom markup like the following:
https://resources.jointjs.com/docs/jointjs/v3.5/joint.html#dia.Cell.markup

```js
markup: [{
        tagName: 'rect',
        selector: 'body'
     }, {
         tagName: 'rect',
         selector: 'header',
     }, {
        tagName: 'text',
        selector: 'headerText',
        className: 'headerLabel'
     }, {
        tagName: 'text',
        selector: 'headerSubText',
        className: 'headerSubLabel'
     }, {
        tagName: 'text',
        selector: 'bodyText',
        className: 'bodyLabel'
 }]
```

Then you can style the sub label how you want in the shape attributes:

```js
attrs: {
   // other attributes
   headerSubText: {
         text: 'headerSubLabel',
         fill: 'black',
         fontFamily: 'Roboto Condensed',
         fontWeight: 'Normal',
         fontSize: 11,
         y: 20,
         x: 10
    }
}
```
